src/site/xdoc/userguide/overview.xml

```diff
@@ -46,6 +46,10 @@
         JNDIConfiguration, you are sheltered from the mechanics of actually retrieving the
         configuration values. These various sources include:
         <ul>
+          <li>
+              <strong>EnvironmentConfiguration</strong>
+              Reads the plattform specific environment variables.
```

Comment:
s/plattform/platform/

Reply:
Sorry for the typo. @garydgregory merged it already. Should I create a new PR?

Reply:
Yep.